Rakefile

```diff
@@ -13,8 +13,6 @@ Rake::TestTask.new(:test) do |t|
   t.libs << 'lib'
   t.libs << 'test'
   t.pattern = ENV['TEST_PATTERN'] || 'test/**/*_test.rb'
-  t.verbose = ENV['TEST_VERBOSE'] == '1'
-  t.warning = true
```

Comment:
Alright, I'm happy to leave those in. It just seems like the Travis logs are full of warnings from gems, for example: https://travis-ci.org/reactjs/react-rails/jobs/49538440#L312